dot/network/helpers_test.go

```diff
@@ -90,7 +90,10 @@ func (s *testStreamHandler) readStream(stream libp2pnetwork.Stream,
 			return
 		} else if err != nil {
 			logger.Debugf("failed to read from stream using protocol %s: %s", stream.Protocol(), err)
-			_ = stream.Close()
+			err := stream.Close()
+			if err != nil {
+				logger.Warnf("failed to close stream: %w", err)
```

Comment:
`%w` is to **w**rap errors with `fmt.Errorf`, for logging it's not a valid verb. Use `%s` instead (stringify the error).

```suggestion
				logger.Warnf("failed to close stream: %s", err)
```